PLEP-0007.rst

```diff
@@ -0,0 +1,189 @@
+| :scope:                                                                      |
+|   A collection of functions, classes, and sub-packages focused on providing  |
+|   analysis techniques that can be applied to data collected from a variety   |
+|   of sources (simulation, experimental, space, etc.).                        |
+|                                                                              |
+|   The focus of an analysis routine should be made as broad as possible.      |
+|   When the routine's functionality becomes highly-tailored to its            |
+|   application, then it should be placed into an appropriate explicit         |
+|   namespace.                                                                 |
++------------------------------------------------------------------------------+
+| **diagnostics**                                                              |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/diagnostics/``                                      |
+| :access: ``plasmapy.diagnostics``                                            |
+| :sub-package PLEP:                                                           |
+| :scope:                                                                      |
+|   | blah blah blah                                                           |
+|   | blah blah blah                                                           |
++------------------------------------------------------------------------------+
+| **dispersion**                                                               |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/dispersion/``                                       |
+| :access: ``plasmapy.dispersion``                                             |
+| :sub-package PLEP:                                                           |
+| :scope:                                                                      |
+|   | blah blah blah                                                           |
+|   | blah blah blah                                                           |
++------------------------------------------------------------------------------+
+| **examples**                                                                 |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/examples/``                                         |
+| :access: ``plasmapy.examples``                                               |
+| :sub-package PLEP:                                                           |
+| :scope:                                                                      |
+|   | blah blah blah                                                           |
+|   | blah blah blah                                                           |
```

Comment:
I've actually been thinking that it may be a good idea to move the examples out of the plasmapy directory and into the main repository; after all, they're not "importable" code and there is no explicit need to keep them there.

Reply:
Agreed.  I've removed the ``examples`` package from this PLEP and opened issue PlasmaPy/PlasmaPy#745.